### PR TITLE
JS-Restaurant Pull Request

### PR DESCRIPTION
**Pull request by alexisbec**
This is a Restaurant landing page, where you can watch the information about the restaurant, all the menu, and a simple contact view, all of these are generated with JavaScript, and the styles are made with CSS.

### CODE REVIEW

**Review by MkrtichSargsyan — CHANGES_REQUESTED** [review]
## _Status:_ Changes Requested :recycle:

## Hello Guys

THIS IS VERY IMPRESSIVE WORK. YOU HAVE DONE A GOOD JOB :+1:

## To Highlight :tada:

1. The project solves the problem it was supposed to solve.
2. Your PR has a good title and description.
3. You have descriptive git commit messages.


However, there are some changes that you need to make in order to get your project approved

Please also check my comments below

- [x] There is no live demo link

- [x] You should use ES6 syntax (arrow functions instead of the function keyword) ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/restaurant_page/es6_requirement.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59622983/))

- [x] Kindly remove the scripts from the head tag of ```dist/index.html```

<hr>

## OPTIONAL


- [ ] You can use a file loader for leading images.
    1. Install [file-loader](https://v4.webpack.js.org/loaders/file-loader/)
    2. Add the loader to your webpack rules
So it will create a new ```imgs``` folder inside your ```dist```  and will add all your images there.

      ex.

      {
        test: /\.(png|svg|jpg|jpeg|gif)$/i,
        use: {
          loader: 'file-loader',
          options: {
            name: '[name].[hash].[ext]',
            outputPath: 'imgs',
          },
        },
      },


Please, make the REQUIRED changes and submit your request again! :grinning:
Good luck. 

> ps: . If you have any questions about my review or need help, you can message me on slack ( @mkrtichsargsyan )

File: dist/index.html [hunk 1]
```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8" />
+    <title>Bolly Restaurant</title>
+    <script src="./content.bundle.js"></script>
+    <script src="./home.bundle.js"></script>
+    <script src="./menu.bundle.js"></script>
+    <script src="./contact.bundle.js"></script>
```
Comment (MkrtichSargsyan) [inline dist/index.html]: Kindly remove these scripts

File: src/contact.js [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+  return contact;
+}
+
+function contactContent() {
+  const main = document.querySelector('#main');
+  main.innerHTML = '';
+  main.appendChild(contact());
+
+  return main;
+}
```
Comment (MkrtichSargsyan) [inline src/contact.js]: Use arrow functions

**Review by oloomoses — APPROVED** [review]
Hello @alexisbec,
**Well done on implementing the changes suggested by the @MkrtichSargsyan**
### Highlights
- Project is deployed and a link provided in the readme :heavy_check_mark:
- Use of ES6 sintax :heavy_check_mark:
- Descriptive pr and readme :heavy_check_mark:
- Linters configuration :heavy_check_mark:

**This project meets all the requirements and therefore:**
## Approved :tada: :muscle: :clap:

##### Congratulations once again and best wishes on the next project.

### Happy coding

_Slack: Oloo Moses_

**Pull request merged**